index.md

```diff
@@ -7,45 +7,34 @@ landing-page: live
+
+* **What’s the difference between the sandbox and production environments?** BCDA’s production environment provides actual patient claims data on an ACO’s prospectively assigned or assignable beneficiaries, while the sandbox uses the same API structure but contains no beneficiary PII or PHI.
+
+* **Why should my ACO try the sandbox first?** ACOs who have gotten used to working in the sandbox have found it easier to make the switch to the production environment. Since beneficiary claims data must be treated in accordance with HIPAA regulations, the sandbox environment provides a lower-stress environment for retrieving data from the API before touching actual patient data.
+
+* **What if my ACO participates in a CMS Innovation Center model?** BCDA’s initial goal is to provide bulk beneficiary claims data that meets the needs of Shared Savings Program ACOs. In the months ahead, BCDA’s research team will begin learning about the data needs of Innovation Center Model ACOs, in order to begin accommodating those ACOs in the API in the future. While the sandbox environment focuses on the needs of Shared Savings Program ACOs, all ACOs and their vendor partners are invited to explore the sandbox and give their feedback.
+
+* **Note:** Some data fields have not yet been mapped from CCLFs to BCDA; The Blue Button API team is working hard to map all of the CCLF fields in the near future.
 
 ## Getting Started
   Moving from the CCLF file structure to BCDA’s FHIR-based API is a fairly big change. BCDA provides documentation and user guides to help Shared Savings Program ACOs at any stage of technical understanding learn how to interact with the API and the data.
 
-  * **If BCDA is going to be your first API:** start with the guide to Getting Started in Sandbox [Getting Started in Sandbox](/sandbox/user-guide/index.html){:target="_blank"}.
-  * **If you’ve worked with APIs before:** start with the [Sandbox Technical Setup Guide.](/sandbox/technical-user-guide/index.html)
-  * **To learn more about the new data structure BCDA will deliver:** review the [Guide to Working with BCDA Data](./data_guide.html){:target="_blank"}.
-  * **To join the queue for an invitation to the production environment:** send an e-mail with your name and your SSP ACO’s name, ID number, and track to [bcapi@cms.hhs.gov](mailto:bcapi@cms.hhs.gov). We will onboard a few ACOs at a time in the order in which requests are received.
+  * **If BCDA is going to be your first API:** start with the guide to [Getting Started in Sandbox](/sandbox/user-guide/){:target="_self"}.
+  * **If you’ve worked with APIs before:** start with the [Sandbox Technical Setup Guide](/sandbox/technical-user-guide/){:target="_self"}.
```

Comment:
After we finished the index page, we renamed the technical getting started guides. Currently says [Sandbox Technical Setup Guide]; let's change that to [Advanced Sandbox User Guide], maintain the link to the sbx tech guide, and stick a period on the end. (ie [Advanced Sandbox User Guide](/sandbox/technical-user-guide/). )